distribution/bwc/build.gradle

```diff
@@ -105,7 +105,7 @@ bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInfo unreleased
         dependsOn fetchLatest
         doLast {
             String refspec = System.getProperty("bwc.refspec.${bwcBranch}") ?: System.getProperty("tests.bwc.refspec.${bwcBranch}") ?: "${remote}/${bwcBranch}"
-            if (System.getProperty("bwc.checkout.align") != null || System.getProperty("tests.bwc.checkout.align") != null) {
```

Comment:
Is this an unrelated change? I'm curious why we have the tests.bwc.refspec above here as well.

Reply:
This was for backwards compatibility with CI when we renamed this property. Just now remembering to go back and remove this. See https://github.com/elastic/infra/pull/13812.